src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ObservationMapper.java

```diff
@@ -178,6 +179,16 @@ private void mapStatus(final LabResult labResult, final Observation observation)
             observation.setStatus(status);
         }
 
+        private void mapInterpretation(final LabResult labResult, final Observation observation) {
+            // Observation.interpretation = SG18.RSL.7857
+            labResult.getInvestigationResult().ifPresent(investigationResult ->
+                Optional.ofNullable(investigationResult.getDeviatingResultIndicator()).ifPresent(deviation ->
+                    observation.getInterpretation().setText(
+                        deviation.getCode() + " : " + deviation.getDescription())
+                )
+            );
```

Comment:
```suggestion
            labResult.getInvestigationResult()
                .map(LaboratoryInvestigationResult::getDeviatingResultIndicator)
                .map(deviation -> deviation.getCode() + " : " + deviation.getDescription())
                .ifPresent(deviation -> observation.getInterpretation().setText(deviation));
```
`Optional#map` can safely go from something present to being empty.
```
labResult.getInvestigationResult() // Optional<LaboratoryInvestigationResult>
    .map(LaboratoryInvestigationResult::getDeviatingResultIndicator) // Optional<DeviatingResultIndicator>
    .map(deviation -> deviation.getCode() + " : " + deviation.getDescription()) // Optional<String>
    .ifPresent(deviation -> observation.getInterpretation().setText(deviation));
```